src/core/resolvers/tdesign.ts

```diff
@@ -69,14 +69,17 @@ function getSideEffects(importName: string, options: TDesignResolverOptions): Si
   if (fileName === 'check-tag')
     fileName = 'tag'
 
-  if (['time-range-picker', 'time-range-picker-panel'].includes(fileName))
+  if (['time-range-picker', 'time-range-picker-panel', 'time-picker-panel'].includes(fileName))
     fileName = 'time-picker'
 
-  if (['date-range-picker', 'date-range-picker-panel'].includes(fileName))
+  if (['date-range-picker', 'date-range-picker-panel', 'date-picker-panel'].includes(fileName))
     fileName = 'date-picker'
 
   if (['color-picker', 'color-picker-panel'].includes(fileName))
     fileName = 'color-picker'
+  
```

Comment:
Please format the file
`pnpm run lint --fix`

Reply:
fixed :)